haskell/stack/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM heroku/cedar:14
+MAINTAINER Gabriel Berke-Williams <gabe@thoughtbot.com>
+
+ENV LANG C.UTF-8
+
+RUN apt-get update && \
+    apt-get install -y --force-yes wget && \
+    wget -q -O- https://s3.amazonaws.com/download.fpcomplete.com/debian/fpco.key | apt-key add - && \
+    echo 'deb http://download.fpcomplete.com/debian/jessie stable main'| tee /etc/apt/sources.list.d/fpco.list && \
+    apt-get update && \
+    apt-get install -y --force-yes stack
+
+# stack stores binaries in /root/.local/bin
+ENV PATH /app/.local/bin:/root/.local/bin:/usr/bin:$PATH
+
+WORKDIR /app
```

Comment:
I think you have to `RUN mkdir /app` for this to actually work.

Also, `heroku docker:release` looks in `/app/user` for build artifacts, so that may be a better working directory.